web/src/components/TestCard/useOnDeleteCallback.tsx

```diff
@@ -0,0 +1,14 @@
+import {MenuInfo} from 'rc-menu/lib/interface';
```

Comment:
Just a small comment, you don't have to add the .tsx extension if the file doesn't explicitly use JSX